design-library/source/_patterns/01-atoms/04-blog-tag/blog-tag~bluebox.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
See previous comments on the variations. I like the naming convention you used for `~bluebox` instead of the `~1` and `~2` in my previous comment